### DISCUSSION TITLE
Dependency between 2 differently scheduled flows

### DISCUSSION THREAD

**Discussion by tri97nguyen** [PrefectHQ/prefect#15929] [Q&A]
Hi, I have 

- job A runs at 1am daily
- job B runs at 3am Monday, Tuesday, Wednesday
- Job B depends on data from Job A (A pulls in some files on the disk)

Job A/B could either Flow/Task, but I think Flow is best because I could schedule it.

_**My question is, how do I see the dependency between Job A and Job B?**_

I cannot create a parent flow and wrap Job A & Job B together, because Job B doesn't always run when Job A runs. So, I cannot see the dependency like that.
How should I approach this? Thank you!

**Comment by zzstoatzz** [comment]
hi @tri97nguyen - as described it seems like there isn't an _explicit_ dependency between A and B, its just that the flow code in B happens to read from some 3rd party source of data that A populates - is that right?

in general, we are interested in (and actively working on) being able to "zoom out" and see the event that triggered a flow run and similar types of big-picture dependencies.

Can you talk a little about what your ideal DX would be here?
